### ISSUE TITLE
comparison of attacks

### ISSUE THREAD

**Issue by smhjafari** [ecksun/Contiki-IDS#2]
Hi,
I ran some attacks code like sinkhole and clone.
now I want to Damaging effect of this attacks for example on energy consumption or memory.
I can not get result by powertrace on this code.

Can you help me how can I comparison these attacks?

thanks

**Comment by ecksun** [comment]
"this code" is very generic, you have to be much more specific, what exactly are you doing and what is not working?

This work is several years old, done towards an outdated version of Contiki. If you are interested in using this work you should probably port it to the latest version of Contiki first (I would gladly accept such a contribution). Is any work done towards such an outdated version of Contiki relevant at all? What versions of Contiki are still being maintained?

**Comment by smhjafari** [comment]
Thanks.

I used version 2.7 of Contiki for running this code.
I want to run specially some code like sinkHole attack and CloneID attack and I want to comparison these code on  power consumption or memory.
So I want a method for doing this work. because when I run ecksun code pwer consuption anf memory consumption didn't respond me.
I want to know does my purpose have a good method like power consution or any other method?

thanks

**Comment by smhjafari** [comment]
Also I want to calculate Power consumption for Attack sinkhole
and I ran this simulate code. but I couldn't receive any result in collect view on power consumption.

Can you help me?

**Comment by smhjafari** [comment]
Can you help me in seeing collect view on power consumption in ecksun Sinkhole attack code?

**Comment by ridhimasen474** [comment]
hi smhjafari,
I am also working in this area. can you please tell me how do you run sinkhole attack and other simulations. It is using some "evil"  folder. I am unable to find that folder.. can you please help me..
Thanks in advance.

**Comment by smhjafari** [comment]
Hi.
You should creater this folder in that path for running of code.

thanks

**Comment by ridhimasen474** [comment]
smhjafari thanks for your reply. can you please share your contact details. 
 my mail id is: ridhimasen474@gmail.com

**Comment by ecksun** [comment]
> Can you help me?

Frankly, probably not, it was a long time ago I did this work. 

> Can you help me in seeing collect view on power consumption in ecksun Sinkhole attack code?

For power consumption I think I simply measured the time spent doing the various activities (sleeping, sending, receiving, processing) and using the data sheet of the hardware in question.

> It is using some "evil" folder

Yes, this project assumes there is a second copy of the repository checked out with a suffix of `-evil`.

A small makefile which clones/creates the correct folders with the correct branches would probably be useful, feel free to contribute.

I will close this issues as its not clear to me what the actual issue is.
